hawkbit-rest-core/src/test/java/org/eclipse/hawkbit/rest/util/JsonBuilder.java

```diff
@@ -33,6 +35,62 @@
  */
 public abstract class JsonBuilder {
 
+    public static String ids(final Collection<Long> ids) {
+        final JSONArray list = new JSONArray();
+        for (final Long smID : ids) {
+            list.put(new JSONObject().put("id", smID));
+        }
+
+        return list.toString();
+    }
+
+    public static String controllerIds(final Collection<String> ids) {
+        final JSONArray list = new JSONArray();
+        for (final String smID : ids) {
+            list.put(new JSONObject().put("controllerId", smID));
+        }
+
+        return list.toString();
+    }
+
+    public static String tags(final List<Tag> tags) throws JSONException {
+        final StringBuilder builder = new StringBuilder();
+
+        builder.append("[");
+        int i = 0;
+        for (final Tag tag : tags) {
+            try {
+                createTagLine(builder, tag);
+            } catch (final Exception e) {
+                e.printStackTrace();
```

Comment:
Why not throwing an exception here? It's obviously wrong JSON. It might be easier to find when you stop processing the tags here instead of waiting for the test to fail due to missing tags later